src/main/java/bookstore/service/impl/UserServiceImpl.java

```diff
@@ -29,7 +29,7 @@ public UserResponseDto register(UserRegistrationRequestDto request) {
         }
         User user = userMapper.toUser(request);
         user.setRoles(Collections.singleton(roleRepository
-                .findRoleByName(Role.RoleName.ROLE_USER)));
+                .findRoleByName(Role.RoleName.ROLE_ADMIN)));
```

Comment:
Why did you change role here? Should each user have an admin role?

Reply:
Anyway, I'm changing roles in DB